client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1821,10 +1821,10 @@ Undocumented.prototype.isSiteImportable = function ( site_url ) {
 	);
 };
 
-Undocumented.prototype.fetchImporterState = function ( siteId ) {
+Undocumented.prototype.fetchImporterState = function ( siteId, fn ) {
```

Comment:
Seems like we don't actually need the callback here?

Reply:
That's true, we don't need it. I added it only temporarily when debugging a failing unit test. Will revert this part.